plugin/core/sessions.py

```diff
@@ -242,21 +242,17 @@ def __init__(self,
         self._initialize()
 
     def has_capability(self, capability: str) -> bool:
-        return capability in self.capabilities and self.capabilities[capability] is not False
+        value = self.get_capability(capability)
+        return value is not False and value is not None
```

Comment:
I remember we had to account for `registerOptions` not being present. In that case I think we place a `None` value in `self.capabilities`?

```
 set_dotted_value(self.capabilities, capability_path, registration.get("registerOptions"))
```

Perhaps this should become:


```
 set_dotted_value(self.capabilities, capability_path, registration.get("registerOptions", {}))
```
to avoid scratching our heads in the future.

Reply:
Good point. Will change.